docker-compose.yml

```diff
@@ -0,0 +1,69 @@
+version: '3'
+
+services:
+  database:
+    container_name: signsfive_database
+    image: mariadb
+    volumes:
+      - ./db_data:/var/lib/mysql
+    restart: always
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_DATABASE: signsfive_dev
+      MYSQL_USER: signsfive_dev
+      MYSQL_PASSWORD: signsfive_dev
+    networks:
+      signsfive_net:
+        ipv4_address: 172.28.1.1
+
+  api:
+    container_name: signsfive_api
+    depends_on:
+      - database
+    image: node:alpine
+    user: node
+    working_dir: /home/node/app
+    environment:
+      - NODE_ENV=development
+      - PORT_API=8282
+    volumes:
+      - ./db:/home/node/app/db
+      - ./node_modules:/home/node/app/node_modules
+      - ./scripts:/home/node/app/scripts
+      - ./src:/home/node/app/src
+      - ./.npm:/home/node/.npm
```

Comment:
do we need a `.npm` volume? it's in the `.gitignore`

Reply:
Not necessary but if want able to view the log, this is easiest way unless you want to use `docker exec bash`